http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -154,11 +154,9 @@ class DataprocInterpreter[F[_]: Parallel](
           .compile
           .drain
 
-        resourceConstraints <- getClusterResourceContraints(params.runtimeProjectAndName,
-                                                            machineConfig.masterMachineType,
-                                                            machineConfig.region
+        jupyterResourceConstraints = RuntimeResourceConstraints(
+          MemorySize(config.dataprocConfig.jupyterReservedMemory.map(_.bytes).getOrElse(MemorySize.fromGb(1.5).bytes))
```

Comment:
Shouldn't this 1.5 be read from Config instead of being hardcoded?

Reply:
It's trying to read from config, Scala reads it in as optional, but if it can't find it in config it will default to 1.5.

Reply:
Oh boy, I need to get back to drinking coffee after lunch so sorry 🤦‍♀️